api/utilities.py

```diff
@@ -0,0 +1,200 @@
+
+    result = {"result": Codes.SUCCESS.value}
+    return session['username'], command, result
+
+
+async def _check_if_data_correct(data: dict[str]):
+    """Check if data is ok with the requirements"""
+    password = data['password']
+    user_by_username = await User.get_user_by_username(data['username'])
+    user_by_email = await User.get_user_by_email(data['email'])
+    if user_by_username or user_by_email:
+        raise LoginError(LoginErrorMessage.USER_ALREADY_EXIST.value)
+    if password != data['repeated_password']:
+        raise PasswordError(PasswordErrorMessage.UNMATCHED_PASSWORD.value)
+    elif not re.search(PASSWORD_SYMBOLS_REQUIREMENTS_PATTERN, password) or not re.match(
+            PASSWORD_COMPOUNDS_REQUIREMENTS_PATTERN,
+            password,
+    ):
+        raise PasswordError(PasswordErrorMessage.INELIGIBLE_PASSWORD.value)
+    elif not re.match(LOGIN_COMPOUNDS_REQUIREMENTS_PATTERN, data['username']):
+        raise LoginError(LoginErrorMessage.INELIGIBLE_LOGIN.value)
+
+
+def login_required(handler: typing.Callable[[web.Request], typing.Awaitable[web.Response]]):
+    """Check for authorization to process request"""
+
+    @wraps(handler)
+    async def wrapper(request: web.Request) -> web.Response:
+        session = await get_session(request)
+        if session.new:
+            session.invalidate()
+            raise web.HTTPUnauthorized(text='Requires authorization')
+        return await handler(request)
+
+    return wrapper
+
+
+def json_required(handler: typing.Callable[[web.Request], typing.Awaitable[web.Response]]):
+    """Check for correct request content to process request"""
+
+    @wraps(handler)
+    async def wrapper(request: web.Request) -> web.Response:
+        if 'Content-type' not in request.headers or request.headers['Content-type'] != 'application/json':
+            raise web.HTTPBadRequest(text='Incorrect request content')
+        return await handler(request)
+
+    return wrapper
+
+
+def create_json_response(data: dict[typing.Union[str, int]]) -> web.Response:
+    """Json response using ujson.dumps"""
+    return web.json_response(data, dumps=ujson.dumps)
+
+
+def _hash_password(password: str) -> str:
+    """Hash password with salt"""
+    hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+    return hashed_password.decode('utf-8')
+
+
+def check_for_required_info_for_login(data: dict[str]):
+    if RequiredData.USERNAME.value not in data.keys() or RequiredData.PASSWORD.value not in data.keys():
+        raise web.HTTPBadRequest(text='Incorrect data')
+
+
+def check_for_required_info_for_registration(data: dict[str]):
+    if RequiredData.EMAIL.value not in data.keys() or RequiredData.USERNAME.value not in data.keys() \
+            or RequiredData.PASSWORD.value not in data.keys() \
+            or RequiredData.REPEATED_PASSWORD.value not in data.keys():
+        raise web.HTTPBadRequest(text='Incorrect data')
```

Comment:
```suggestion
    if (
        RequiredData.EMAIL.value not in data.keys()
        or RequiredData.USERNAME.value not in data.keys()
        or RequiredData.PASSWORD.value not in data.keys()
        or RequiredData.REPEATED_PASSWORD.value not in data.keys()
    ):
        raise web.HTTPBadRequest(text='Incorrect data')
```
Рекомендую придерживаться такого кодстайла